exchanges/kraken/kraken_wrapper.go

```diff
@@ -60,9 +60,10 @@ func (k *Kraken) SetDefaults() {
 		RequestFormat: &currency.PairFormat{
 			Uppercase: true,
 			Separator: ",",
+			Delimiter: "",
 		},
 		ConfigFormat: &currency.PairFormat{
-			Delimiter: "-",
+			Delimiter: "",
```

Comment:
This will introduce a pair splitting issue as currency.NewPairFromString will split the string [0:3] and [3:] by default which means that pairs like QTUMEUR will end up becoming QTU and MEUR, which is why the adjustment below was done

Reply:
Yeah looks like Bitfinex has this problem too. I'll fix both